src/families/tron/bridge/js.js

```diff
@@ -259,20 +239,54 @@ const currencyBridge: CurrencyBridge = {
   scanAccountsOnDevice
 };
 
-const createTransaction = a => {
-  throw new CurrencyNotSupported("tron currency not supported", {
-    currencyName: a.currency.name
-  });
-};
+const createTransaction = () => ({
+  family: "tron",
+  amount: BigNumber(0),
+  recipient: "",
+  networkInfo: null
+});
 
 const updateTransaction = (t, patch) => ({ ...t, ...patch });
 
-const getTransactionStatus = a =>
-  Promise.reject(
-    new CurrencyNotSupported("tron currency not supported", {
+const getTransactionStatus = async (a, t) => {
+  const errors = {};
+  const warnings = {};
+  const tokenAccount = !t.subAccountId
+    ? null
+    : a.subAccounts && a.subAccounts.find(ta => ta.id === t.subAccountId);
+  const account = tokenAccount || a;
+
+  const estimatedFees = BigNumber(0); //TBD
```

Comment:
is it gas limit times gas price?

Reply:
So Tron use the a system call bandwidth, there's only two cases that doesn't use any bandwidth and use tron.
To sum up : 
-If you don't have any bandwidth it will cost around 0.02 TRX
- Like Tezos, if there's no transaction u need to activate this account and it will cost around 0.1 TRX
It's just the case that I found may i missed some. I'll try to implement it